meilisearch-http/tests/assets/backups/v1/test/settings.json

```diff
@@ -0,0 +1,59 @@
+{
+    "rankingRules": [
+        "typo",
+        "words",
+        "proximity",
+        "attribute",
+        "wordsPosition",
+        "exactness"
+    ],
+    "distinctAttribute": "email",
+    "searchableAttributes": [
+        "balance",
+        "picture",
+        "age",
+        "color",
+        "name",
+        "gender",
+        "email",
+        "phone",
+        "address",
+        "about",
+        "registered",
+        "latitude",
+        "longitude",
+        "tags"
+    ],
+    "displayedAttributes": [
+        "id",
+        "isActive",
+        "balance",
+        "picture",
+        "age",
+        "color",
+        "name",
+        "gender",
+        "email",
+        "phone",
+        "address",
+        "about",
+        "registered",
+        "latitude",
+        "longitude",
+        "tags"
+    ],
+    "stopWords": [
+        "in",
+        "ad"
+    ],
+    "synonyms": {
+        "wolverine": ["xmen", "logan"],
+        "logan": ["wolverine", "xmen"]
+    },
+    "attributesForFaceting": [
+        "gender",
+        "color",
+        "tags",
+        "isActive"
+    ]
+}
```

Comment:
Could you add a newline to all of the file that miss one at the end?

(And setup your editor too 😄)